### ISSUE TITLE
add function to subset vector of variant IDs to a specified chromosome

### ISSUE THREAD

**Issue by broomej** [Blue-Lab/blPipeline#4]
Function should take a vector of IDs, a GDS connection object and a chromosome number, and return a vector that is just the variants from the specified chromosome. Example function:

```
subset_var_by_chrom <- function(variant_id, chrom, gds) {
  chrom_map <- data.frame(
    id = SeqArray::seqGetData(gds, "variant.id"),
    chromosome = SeqArray::seqGetData(gds, "chromosome")
  )
  chrom_map <- dplyr::filter(chrom_map, chromosome == chrom)
  var_id_out <- variant_id[variant_id %in% chrom_map$id]
  return(var_id_out)
}
```

could be modified to take a vector or range of chromosomes e.g. `c(14, 19)` or `1:9`

**Comment by broomej** [comment]
I was able to add some tests using `gdsfmt::createfn.gds()`, but this is my first time manually manipulating GDS objects like this so just wanted to note that here in case it looks funny. I did get it to work in the testing framework though

**Comment by broomej** [comment]
implemented in 7f554578b025e1c650d869fecf14b0b298c1a8d5
